.github/actions/setup-test-cluster/action.yaml

```diff
@@ -0,0 +1,219 @@
+name: Set Up Test Cluster
```

Comment:
Created this reusable workflow since we're using it (setting up a test cluster) for both the regular e2e tests as well as the SDK e2e tests.